py/selenium/webdriver/chrome/service.py

```diff
@@ -24,7 +24,7 @@ class Service(service.ChromiumService):
     """
 
     def __init__(self, executable_path, port=0, service_args=None,
-                 log_path=None, env=None):
+                 log_path=None, env=None, creationflags=0):
```

Comment:
I would rather avoid having a new item added to the method signature. Could you have this as a property in these classes and then when `start` is called we check if it is populated and used.

Also, there is no documentation added in the docstring for these changes. It does not explain what the 0 means to people who may come across it.

Reply:
The `0` is the default value of a Python subprocess, maybe using a comment will be helpful.

I assume that after doing what you asked, the above sample code (to use this new feature), will become like this:
```
from selenium import webdriver
from selenium.webdriver.chrome.service import Service as ChromeService # Similar thing for firefox also!
from subprocess import CREATE_NO_WINDOW # This flag will only be available in windows

chrome_service = ChromeService('chromedriver')
chrome_service.creationflags = CREATE_NO_WINDOW # We change the property in Service class, default was 0
driver = webdriver.Chrome(service=chrome_service)
```
Is that what you mean? If yes, then I will proceed with this work:
- Add a property `self.creationflags=0` in the `Service` class constructor.

Reply:
That's what I mean. The other `__init__` signatures grew when I wasn't watching and it became messy. I would rather try keep them clean if possible

Reply:
One more thing I would like to mention. When the `subprocess.Popen(...)` is called, there is a default argument `creationflags=0` in the `Popen(...)` function in the official python subprocess module. (Even if we do not provide this argument!)